src/static/js/webmentions.js

```diff
@@ -0,0 +1,209 @@
+
+      reactionDIVMeta.setAttribute("class", "webmention-meta");
+
+      const reactionTIME = document.createElement("time");
+      reactionTIME.setAttribute("class", "webmention-pub-date");
+      reactionTIME.setAttribute("datetime", reaction["published"]);
+      reactionTIME.textContent = formatDate(reaction["published"]);
+
+      const reactionSPAN = document.createElement("span");
+      reactionSPAN.setAttribute("class", "webmention-divider");
+      reactionSPAN.setAttribute("aria-hidden", "true");
+      reactionSPAN.textContent = " ⋅ ";
+
+      const reactionASource = document.createElement("a");
+      reactionASource.setAttribute("class", "webmention-source");
+      reactionASource.setAttribute("href", reaction["url"]);
+      reactionASource.textContent = document.querySelector(".reactions").getAttribute("data-source");
+
+      reactionDIVMeta.appendChild(reactionTIME);
+      reactionDIVMeta.appendChild(reactionSPAN);
+      reactionDIVMeta.appendChild(reactionASource);
+    }
+
+    reactionLI.appendChild(reactionA);
+    if (reactionType === "replies" || reactionType === "mentions") {
+      reactionLI.appendChild(reactionDIVContent);
+      reactionLI.appendChild(reactionDIVMeta);
+    }
+    webmentionReactionsList.appendChild(reactionLI);
+  });
+  document.querySelector(`#${reactionType}-panel`).appendChild(webmentionReactionsList);
+}
+
+// Parses and renders mentions into likes, reposts, replies and mentions
+function renderWebmentions(webmentions) {
+  if (!webmentions.length) {
+    return;
+  }
+
+  renderReactions(webmentions, "likes", "like-of");
+  renderReactions(webmentions, "reposts", "repost-of");
+  renderReactions(webmentions, "replies", "in-reply-to");
+  renderReactions(webmentions, "mentions", "mention-of");
+}
+
+// Process webmention promise
+function processWebmentions(targetURL) {
+  getWebmentions(targetURL)
+    .then(webmentions => renderWebmentions(webmentions))
+    .catch(e => {
+      console.error(e)
+      gtag('event', 'error', { 'event_category': 'webmentions.js', 'event_label': e, 'value': 1 })
+    })
+}
+
+// Change tabs for webmentions UI
+function changeTabs(target) {
+  const parent = target.parentNode;
+  const grandparent = parent.parentNode;
+
+  // Remove all current selected tabs
+  parent
+    .querySelectorAll('[aria-selected="true"]')
+    .forEach(t => t.setAttribute("aria-selected", false));
+
+  // Set this tab as selected
+  target.setAttribute("aria-selected", true);
+
+  // Hide all tab panels
+  grandparent
+    .querySelectorAll('[role="tabpanel"]')
+    .forEach(p => p.setAttribute("hidden", true));
+
+  // Show the selected panel
+  grandparent.parentNode
+    .querySelector(`#${target.getAttribute("aria-controls")}`)
+    .removeAttribute("hidden");
+}
+
+function addTabListeners() {
+  const tabs = document.querySelectorAll('.reactions [role="tab"]');
+  const tabList = document.querySelector('.reactions [role="tablist"]');
+
```

Comment:
Fix the fact it's broken for prior years:

```suggestion

  if (!tabs || !tabList) {
    return;
  }
```